localstack/utils/net.py

```diff
@@ -88,7 +108,7 @@ def is_port_open(
 def wait_for_port_open(
     port: int, http_path: str = None, expect_success=True, retries=10, sleep_time=0.5
 ):
-    """Ping the given network port until it becomes available (for a given number of retries).
+    """Ping the given TCP network port until it becomes available (for a given number of retries).
```

Comment:
Perhaps the functions in this file could all be updated to take `IntOrPort` (or `Port` if they are not part of the public API), then the entire utility file would be protocol agnostic. What do you think? Perhaps an update for a future PR.

Reply:
Good point, agreed. Would probably leave that for a future iteration.. 👍